### DISCUSSION TITLE
Popup CSS values in themes.

### DISCUSSION THREAD

**Discussion by jack3308** [Clooos/Bubble-Card#299] [Feature requests]
I've been trying to create a them that I can apply at the dashboard level so that all of my popup cards are consistent without a heap of editing in each card, but I'm having a hard time with a couple visual elements of the pop-up card. Namely:

- Shader at the top of popups
- The box shadow surrounding popups
- The shadow behind the header on the popups

Just hoping for some direction more than anything, I appreciate this probably isn't an "issue" but just hoping to remove all of those elements to have a clean transparent blur popup.

**Comment by ntallfellow** [comment]
Hmm.. I would probably recommend a template coded to only affect specific parts of the popup.
Otherwise ctrl+F isnt too bad, especially if you use the raw config editor at the parent page
